.github/workflows/ci.yml

```diff
@@ -121,7 +122,8 @@ jobs:
     - name: Build ${{ env.PACKAGE_NAME }} + consumers
       run: |
         python -c "from urllib.request import urlretrieve; urlretrieve('${{ env.BUILDER_HOST }}/${{ env.BUILDER_SOURCE }}/${{ env.BUILDER_VERSION }}/builder.pyz?run=${{ env.RUN }}', 'builder.pyz')"
-        python builder.pyz build -p ${{ env.PACKAGE_NAME }} --target windows-${{ matrix.arch }} --compiler msvc-16
+        python builder.pyz build -p ${{ env.PACKAGE_NAME }} --target windows-${{
+        matrix.arch }} --compiler msvc-16
```

Comment:
trivial: Weird change. Was this on purpose? seems like a weird change to make when the line before this is way longer

if you want to split this across multiple lines, can we do it the traditional way with backslashes? 
```
python builder.pyz build -p ${{ env.PACKAGE_NAME }} \
  --target windows-${{ matrix.arch }} --compiler msvc-16
```

Reply:
ah, i was experimenting with some flags on the ci runs and i think my ide decided to split it into 2 lines. let me revert it